tensorflow_encrypted/protocol/pond.py

```diff
@@ -1538,24 +1544,24 @@ def _sum_core(prot: Pond,
 
 def _sum_public(prot: Pond,
                 x: PondPublicTensor,
-                axis: int,
-                keepdims: Optional[bool]) -> PondPublicTensor:
+                axis: Optional[int] = None,
+                keepdims: Optional[bool] = None) -> PondPublicTensor:
     y_on_0, y_on_1 = _sum_core(prot, x, axis, keepdims)
     return PondPublicTensor(prot, y_on_0, y_on_1, x.is_scaled)
 
 
 def _sum_private(prot: Pond,
                  x: PondPrivateTensor,
-                 axis: int,
-                 keepdims: Optional[bool]) -> PondPrivateTensor:
+                 axis: Optional[int] = None,
+                 keepdims: Optional[bool] = None) -> PondPrivateTensor:
     y_on_0, y_on_1 = _sum_core(prot, x, axis, keepdims)
     return PondPrivateTensor(prot, y_on_0, y_on_1, x.is_scaled)
 
 
 def _sum_masked(prot: Pond,
                 x: PondMaskedTensor,
-                axis: int,
-                keepdims: Optional[bool]) -> PondPrivateTensor:
+                axis: Optional[int] = None,
+                keepdims: Optional[bool] = None) -> PondPrivateTensor:
     # y_on_0, y_on_1 = _sum_core(prot, x.unmasked, axis, keepdims)
     # return PondPrivateTensor(prot, y_on_0, y_on_1)
     raise NotImplementedError
```

Comment:
for masked values, just forward to the unmasked values: `prot.sum(x.unmasked, .., ..)`